### DISCUSSION TITLE
Model texture changes on Quick-look

### DISCUSSION THREAD

**Discussion by tachnazarov** [google/model-viewer#3124] [Q&A]
### Description
The model look as expected when viewing on browser but when I place the model in AR, the model looks so different. It happens for some models, while most of them are showing properly. PLease see example in glitch and screenshots:

![IMG_0915](https://user-images.githubusercontent.com/90166785/149304082-f675779e-7bc7-4a8e-8ae9-dece9c976083.PNG)
![IMG_0912](https://user-images.githubusercontent.com/90166785/149304134-01912889-f702-4d1b-83a1-a5f9ca133dfa.PNG)
<img width="1440" alt="Screen Shot 2565-01-12 at 18 31 55" src="https://user-images.githubusercontent.com/90166785/149304137-ac24d043-2721-4432-a71a-f65252df2143.png">

#### Live Demo
https://abrasive-puzzled-smoke.glitch.me

### Version
- model-viewer: v1.10.0

### Browser Affected

- [ ] Chrome, version: xx.x.xxxx.xx
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [x] Safari

### OS

- [x] Android
- [x] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [ ] SceneViewer
- [x] QuickLook

**Comment by elalish** [comment]
Our on-the-fly USDZ conversion is not perfect. For any models  that you  find to look significantly different in Quick Look, I'd recommend finding another tool to generate USDZ and then serve it separately as the `ios-src`. Keep in mind that AR will tend to look a bit different anyway since it uses different (estimated) lighting.

**Reply by elalish** [reply]
Speaking of rendering inconsistency @grorg, we get a lot like this where we can't reliably convert a glTF to a USDZ and retain the rendered look. This is why having a universally-supported format is so important, and why glTF is called the JPEG of 3D.
